packages/cardano-services/test/StakePool/DbSyncStakePoolProvider/StakePoolBuilder.test.ts

```diff
@@ -279,4 +301,38 @@ describe('StakePoolBuilder', () => {
       expect(result).toMatchSnapshot();
     });
   });
+  describe('queryPoolAPY', () => {
+    describe('sort', () => {
+      it('by default sort (APY desc)', async () => {
+        const result = (await builder.queryPoolAPY([1, 15])).map(({ apy }) => ({ apy }));
+        expect(result).toHaveLength(2);
+        expect(result[0].apy).toBeGreaterThan(result[1].apy);
+        expect(result).toMatchSnapshot();
+      });
+      it('by APY asc', async () => {
+        const result = (await builder.queryPoolAPY([1, 15], { sort: { field: 'apy', order: 'asc' } })).map(
+          ({ apy }) => ({ apy })
```

Comment:
```suggestion
          ({ apy }) => ({ apy })
```
Should we include a pool identifier to improve the snapshots?

Reply:
This query has the `hashId` as the identifier to match with the rest of the pool information but that could change as it's generated by postgres. We could add a couple of `JOIN`s to the query to get the bech32 pool id but it won't be used anywhere else than in tests.

Reply:
@mkazlauskas thoughts on the presence of the pool ID in the query to facilitate verification? I'm a little concerned since it's `JOIN`ing :thinking:

Reply:
Don't think we should be adding JOINs just for tests. We're already querying by `hashesIds: number[]` as first argument in `builder.queryPoolAPY([1, 15]` so adding it in the snapshot won't make it worse.